### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by achaacha**
Just tweaking a few lines in Post Installation to avoid confusion.

### CODE REVIEW

File: docs/en-US/installation/index.md [hunk 1]
```diff
@@ -33,7 +33,9 @@ There are 2 methods, both involve downloading a zip file:
 
 ## Post installation
 
-Once you've installed everything, will be copied automatically `config/default-config.yml` from `config/config.yml`. Any changes to `default-config.yml` will get overwritten when you update VVV, so always make changes to `config.yml`.  VVV will do this automatically after the first `vagrant up`.
+After installation, a configuration file will be generated: `config/default-config.yml`. **Do not make any changes to this file.** All alterations to the configuration files must go in `config/config.yml` which **WILL BE GENERATED ONLY AFTER RUNNING THE FIRST `vagrant up`.**
```
Comment (Mte90) [inline docs/en-US/installation/index.md]: well the new file is a copy of the default, so can be handled manually, the rest is fine for me

**Comment by tomjn** [comment]
I don't know that this is how we want to solve this problem, people would need to know this before they provision, but this is telling them in the section about post-provision

**Comment by Mte90** [comment]
So you mean inform the user about config.yml before run any vagrant commands?

**Comment by tomjn** [comment]
There’s no point warning them after they provisioned that they should have
done something before they provisioned, it’s a “well why didn’t you say
earlier when I was provisioning?!” thing

**Review by tomjn — APPROVED** [review]
I rewrote things to make more sense, and added one or two tweaks to the surrounding sections

**Pull request merged**